package/astronomer/environment.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+ASTRONOMER_CERTIFIED_VERSION = os.environ.get("ASTRONOMER_CERTIFIED_VERSION", "2.3.2+astro.2")
```

Comment:
A file for this is overkil. Just put this one line where it is needed.

Additionally having a fallback version of 2.3.2 seems odd -- saying "unknown" would be better.